lib/lynx/system_out.rb

```diff
@@ -0,0 +1,14 @@
+require 'open3'
+require 'lynx/error'
+
+module Lynx
+  module SystemOut
+    def self.run(command)
+      stdout, stderr, status = Open3.capture3(command)
+      unless status.success?
+        raise(Lynx::Error, "\n [ERROR] #{stderr} \n [ERROR] Failed to perform: #{command}")
```

Comment:
These don't need `[ERROR]` tags. I might also recommend flipping the ordering here:

```
Failed to perform: command
STDERR: stderr
```